spec/controllers/group_assignment_invitations_controller_spec.rb

```diff
@@ -22,7 +22,7 @@
                              title: 'HTML5',
                              grouping: grouping,
                              organization: organization,
-                             public_repo: false)
+                             public_repo: true)
```

Comment:
I wonder what was the reason for us to use private repo in the previous test cases.

I changed it to `true` as I was not able to create a private repo in a GitHub organization and can not record a VCR of creating a private repo.

Maybe I can figure it out later and fix this.